### DISCUSSION TITLE
Suggestion: Upgrade Grafana in SO2.3.50

### DISCUSSION THREAD

**Discussion by Masaya-A** [Security-Onion-Solutions/securityonion#3866] [Ideas] [answered]
Current SO2.3.40 -> Grafana v7.3.4

Could you update to v7.4.2 or later, please?
https://grafana.com/docs/grafana/latest/release-notes/

**Comment by TOoSmOotH** [comment, marked as answer]
Testing 7.5.4. If it passes we will include it in 2.3.50
